rundeckapp/grails-app/domain/rundeck/ScheduledExecutionStats.groovy

```diff
@@ -2,8 +2,9 @@ package rundeck
 
 import com.fasterxml.jackson.core.JsonParseException
 import com.fasterxml.jackson.databind.ObjectMapper
+import org.rundeck.app.data.model.v1.execution.RdJobStats
 
-class ScheduledExecutionStats {
+class ScheduledExecutionStats implements RdJobStats {
```

Comment:
Check out this PR to see how I've used ScheduledExecutionStats. Add a field. String jobUuid.
https://github.com/rundeck/rundeck/pull/8226/files#diff-bc5003a3910ea811d7bb041050f931420afe24bc6c8899740d276413d7e8ceb4